tests/frameworks/OSXShims/include/windows.h

```diff
@@ -0,0 +1,55 @@
+//******************************************************************************
```

Comment:
I'll file a separate issue for this - I'm scared of going over the file limit.

For reference, we have 44 instances of lowercase, 27 instances of uppercase. However, in unit tests there are only 2 instances of uppercase (soon to be 1) (we may only care about instances in unit tests?)

---
In reply to: [72312429](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72312429) [](ancestors = 72312429)